app/pug/splash.pug

```diff
@@ -1,17 +1,15 @@
-
 doctype html
 
-
 head
   title DUNE APA Construction DB
-  link(rel = "preconnect" href = "https://fonts.gstatic.com")
-  link(href = "https://fonts.googleapis.com/css2?family=Inconsolata&family=Palanquin&display=swap", rel = "stylesheet")
-  link(rel = 'stylesheet' href = '/dist/formiojs/formio.full.min.css')
+  
+  link(rel = 'stylesheet', href = '/css/splash.css')
+  link(rel = 'stylesheet', href = '/dist/formiojs/formio.full.min.css')
+  link(rel = 'stylesheet', href = '/dist/bootstrap/css/bootstrap.min.css')
+  link(rel = 'stylesheet', href = 'https://fonts.googleapis.com/css2?family=Inconsolata&family=Palanquin&display=swap')
+  link(rel = 'preconnect', href = 'https://fonts.gstatic.com')
```

Comment:
Question:  Is there benefit to this prefetch if the fonts are being loaded from a different domain?

Reply:
Honestly have no idea ... it was already like that, so I just left it as is, apart from cleaning up the code itself.  I can dive a bit deeper into it and get back to you.

Reply:
Looks like the `preconnect` line isn't needed, no ... as you say, the font stylesheet is already being loaded in the previous line anyway, and it doesn't appear to make any difference when removed.

There's also an identical situation in `dashboard.pug` - same two lines, with the preconnect one apparently making no difference at all.  I'll remove that one as well.

Reply:
Preconnect statements enable browsers to establish connections to external domains before reading statements to load resources.  It can improve UX.  https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preconnect

Reply:
Right, I did read that page ... didn't fully understand the importance I guess.  So pointing the `preconnect` lines to `fonts.gstatic` doesn't do anything since that's not where the fonts are being loaded from ... they should point to the same address as the `fonts.googleapis` `stylesheet` line?  And the `preconnect` line should go before that one, not after?

Reply:
Looking at it some more, would the `preload` link type be more appropriate here, instead of `preconnect`?  The MDN page for the former specifically uses the example of loading fonts from a CSS as an example of how using `preload` can give performance improvement.

Reply:
Sure, preload makes more sense I think.